apps/src/utils.js

```diff
@@ -1,33 +1,6 @@
 /* global define */
 'use strict';
 
-var savedAmd;
-
-// Do some hackery to make it so that lodash doesn't think it's being loaded
-// via require js
```

Comment:
Maybe @Bjvanminnen or @cpirich can shed some light on why this was ever necessary, and whether it is still likely to be needed?  If it is needed, then we need to make a reverse change and clean up places where we were including lodash directly.  I suspect we're fine without it though - we've had some usages without this wrapper for a while, and test all pass without it.